File: aneis.c
#include <stdio.h>

int main(){
	int tamanho;
	scanf("%d",&tamanho);
	int vetor[19];
	int var;
	for(int i = 0;i<20;i++){
		vetor[i]=0;
	}
	int maior = 0;
	for (int i = 0;i<tamanho;i++){
		scanf("%d",&var);
		vetor[var-11]+=1;
		if(vetor[var-11] > maior){
			maior = vetor[var-11];
		}
	}
	printf("%d\n",maior);



	return 0;
}

File: aneis.py

def main():
	tamanho = int(input())
	dic = {}
	for i in range(tamanho):
		var = int(input())
		if(var in dic):
			dic[var] = dic[var]+1
		else:
			dic[var] = 1
	
	maior = var
	for i in dic:
		if(dic[i]>dic[maior]):
			maior = i
			
	print(dic[maior])


main()
